test/modules/TestResultCoordinator/Reports/CountingReport.cs

```diff
@@ -157,11 +165,6 @@ bool IsPassedHelper()
                         {
                             return ((double)this.TotalMatchCount / this.TotalExpectCount) > .8d;
                         }
-                        // Product issue for messages when broker is enabled.
-                        else if (this.Topology == Topology.Nested && this.MqttBrokerEnabled && this.TestDescription.Contains(MessagesTestDescription))
-                        {
-                            return ((double)this.TotalMatchCount / this.TotalExpectCount) > .99d;
-                        }
```

Comment:
This branch was in the wrong place. Since there are EventHub settings it should be in that match arm above.